client/src/app/appCtrl.js

```diff
@@ -15,8 +15,15 @@ angular.module('app').controller('appCtrl', ['$scope', '$http', '$timeout', 'app
       threshold : appConfig.MAX_FILE_SIZE,
       show : false,
       paused : false,
-      aborted : false
-    }
+      aborted : false,
+    },
+    highlighting : { // section for highlighting (anomalie) over a threshold
```

Comment:
These variables don't need to be in the view object, since they aren't referenced in the templates. They can simply be private variables.